packages/slate-editor/src/modules/popper-options-context/types.ts

```diff
@@ -5,7 +5,7 @@ export interface Context {
     modifiers?: {
         preventOverflow?: PreventOverflowModifier['options'];
     };
-    placement?: 'left' | 'right';
+    placement?: 'left' | 'right' | 'bottom';
```

Comment:
Can we make it work like in the Figma mockups?

1. If the button is left-aligned, the menu is bound to the button bottom left corner.
2. If the button is centred, the menu is bound to the button bottom middle part.
3. If the button is right-aligned, the menu is bound to the button right corner.

![Screenshot from 2023-08-15 11-50-57](https://github.com/prezly/slate/assets/370680/93aaaf53-3731-47b6-ae36-15f7394dd6e2)
![Screenshot from 2023-08-15 11-50-53](https://github.com/prezly/slate/assets/370680/fc13f34a-ee2c-40ae-b634-9a9f304e06b0)
![Screenshot from 2023-08-15 11-50-45](https://github.com/prezly/slate/assets/370680/bf785db8-6dc5-4e18-82fa-9db3f5d6c224)

Reply:
Checking.

Reply:
> ![image](https://github.com/prezly/slate/assets/370680/4aa5d7dc-1f8a-4642-8be1-24f1c5029f5c)

Nope, this is not good enough. Let me see if I can fix it.

Reply:
dfc7ebb6395baa26a083ea90384f3d9a6723fde6

[Screencast from 2023-08-16 16-29-17.webm](https://github.com/prezly/slate/assets/370680/3a53e45f-1cdf-4d58-be7d-ffebcf6b4d8b)

Reply:
Looks good!